ducanhpham0312/COMP.SE.200-2023-2024-1
# ci: setup gha

### Description: 
- Set up CI runs for the repository
- Set up coveralls
- Write first test for isDate
- Add coverage badge for README
### Testing:
- Check the CI run in this PR
- Go to [Coveralls report](https://coveralls.io/github/ducanhpham0312/COMP.SE.200-2023-2024-1), the test coverage report should be visible.
- Local testing: `npm install && npm test`
- Open README under `setup-gha` branch, the badge should be visible with status "unknown". It should be a number after this PR is merged.
